src/features/Movies/Movies.component.tsx

```diff
@@ -1,3 +1,66 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { useAppSelector } from "hooks";
+import { Container } from "react-bootstrap";
+import { useQuery } from "react-query";
 
-export const Movies = () => <div>Movies</div>;
+import { SelectedFiltersType } from "./types";
+import { getMoviesByFilters } from "./Movies.api";
+import { EMPTY_STRING, PAGE_NUMBER_ONE } from "constants/common";
+import { Pagination } from "features/Pagination";
+import { MoviesList } from "features/Dashboard/MoviesList";
+import { getDateString } from "utils";
+import { Filters } from "features/Filters";
+
+export const Movies = () => {
+  const activeLanguage = useAppSelector((state) => state.language.activeLanguage);
+  const [genreId, setGenreId] = useState<string>(EMPTY_STRING);
+  const [selectedFilters, setSelectedFilters] = useState<SelectedFiltersType>(null);
+  const [page, setPage] = useState(PAGE_NUMBER_ONE);
+  const [startDate, setStartDate] = useState<any>(null);
+  const [endDate, setEndDate] = useState<any>(null);
+  const {
+    isSuccess: areMoviesSuccess,
+    data: searchedMoviesData,
+    refetch,
+  } = useQuery(["getMoviesByFilters", activeLanguage, page], () => getMoviesByFilters(activeLanguage, selectedFilters, page), {
+    refetchOnWindowFocus: false,
+    enabled: false,
+  });
+
+  useEffect(() => {
+    if (selectedFilters) refetch();
+  }, [refetch, selectedFilters, activeLanguage, page]);
+
+  const genreChangeHandler = (event: React.ChangeEvent<HTMLSelectElement>) => setGenreId(event.target.value);
```

Comment:
Just to my mind name `genreChangeHandler` is a bit harder to read than just `changeGenre`